cdp_scrapers/instances/houston.py

```diff
@@ -0,0 +1,531 @@
+#     membersTable = peopleTable.find_all("tr")[1]
+#     districtTable = membersTable.find("table").find("tr").find_all("td")
+#     seat = ""
+#     # left and right district
+#     for td in districtTable:
+#         text = td.find("span").find_all("br")
+#         for br in text:
+#             content = br.previousSibling
+#             if type(content) is bs4.element.NavigableString:
+#                 if content.text.strip() == name:
+#                     seat = content.nextSibling.nextSibling.strip()
+#     # lower district
+#     underDistrict = membersTable.find("p").find("span").find("br")
+#     underDName = underDistrict.previousSibling
+#     if underDName.text.strip() == name:
+#         seat = underDName.nextSibling.nextSibling.strip()
+#     # left and right position
+#     positionTable = membersTable.find_all("table")[1].find("tr").find_all("td")
+#     for td in positionTable:
+#         textP = td.find("span").find_all("br")
+#         for br in textP:
+#             contentP = br.previousSibling
+#             if type(contentP) is bs4.element.NavigableString:
+#                 if contentP.text.strip() == name:
+#                     seat = contentP.nextSibling.nextSibling.strip()
+#     # lower position
+#     underPosition = membersTable.find_all("span")[-1].find("br")
+#     underPName = underPosition.previousSibling
+#     if underPName.text.strip() == name:
+#         seat = underPName.nextSibling.nextSibling.strip()
+#     return seat
+
+
+# def get_person(name: str, event: Tag) -> ingestion_models.Person:
+#     """
+#     Get the seat and role for one person
+
+#     Parameters:
+#     -----------------
+#     name:str
+#         The name of one person
+
+#     Returns:
+#     -----------------
+#     ingestion_models.Person:
+#         Seat and role information for one person
+#     """
+#     return ingestion_models.Person(
+#         name=name,
+#         is_active=get_role(name)[1],
+#         seat=ingestion_models.Seat(
+#             name = get_seat(name, event),
+#             roles = ingestion_models.Role(title=get_role(name)[0])
+#         ),
+#     )
+
+
+# missing: get_votes()
+
+
+def remove_extra_type(element: Union[Tag, NavigableString, None]) -> Tag:
+    if isinstance(element, NavigableString) or element is None:
+        raise ValueError(f"Wrong Type {type(element)}")
+    return element
+
+
+def get_body_name(event: Union[Tag, NavigableString, None]) -> str:
+    """
+    Get the body name for an event
+
+    Parameter:
+    ----------------
+    event: Union[Tag, NavigableString, None]
+        All elements in the page that we want to scrape
+
+    Returns:
+    ----------------
+    str
+        The body name
+    """
+    print("start get body name")
+    event = remove_extra_type(event)
+    bodyTable = event.find_all("table")[1].find("table")
+    if "CITY COUNCIL" in bodyTable.text:
+        return "City Council"
+    else:
+        return bodyTable.find_all("span")[3].text
```

Comment:
You dont need this if else you just need to call

`bodyTable.text.title()` to make it "title case"